python-spec/testing/test_collection.py

```diff
@@ -1,18 +1,16 @@
 import unittest
 from typing import Any
 
-from somacore import collection
-from somacore import experiment
-from somacore import measurement
+from somacore import ephemeral
 
 
-class SimpleCollectionTest(unittest.TestCase):
+class EphemeralCollectionTest(unittest.TestCase):
     def test_basic(self):
         # Since the SimpleCollection implementation is straightforward this is
```

Comment:
might want to `s/\bSimple(.+)\b/Ephemeral$1/`

Reply:
Whoops! Thank you.